http/document_service.go

```diff
@@ -140,7 +152,16 @@ func (h *DocumentHandler) handlePostDocument(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
-	if err := encodeResponse(ctx, w, http.StatusCreated, newDocumentResponse(req.Namespace, req.Document)); err != nil {
+	now := h.timeGenerator.Now()
+
+	if err := addLog(ctx, h.KeyValueLog, influxdb.DocumentsResourceType, req.Document.ID,
```

Comment:
I wonder if it might be better to have logging be on a lower layer than the http service (specifically, in `kv`). We don't seem to have documents in our cli right now, but if/when we do, logs won't be created from the cli.

Reply:
I agree, this shouldn't be exclusive the the HTTP service